rotkehlchen/tests/unit/test_tokens.py

```diff
@@ -32,26 +40,43 @@ def test_detect_tokens_for_addresses(tokens):
     USD price queries are mocked so we don't care about the result.
     Just check that all prices are included
     """
-    addr1 = string_to_evm_address('0x8d89170b92b2Be2C08d57C48a7b190a2f146720f')
-    addr2 = string_to_evm_address('0xB756AD52f3Bf74a7d24C67471E0887436936504C')
+    addr1, addr2, addr3 = ethereum_accounts
+    addr3_proxy = string_to_evm_address('0x394C1D68498DEB24AC9F5502DD5450a0353e17dc')
 
+    rotki = rotkehlchen_api_server.rest_api.rotkehlchen
+    tokens = rotki.chains_aggregator.ethereum.tokens
     tokens.evm_inquirer.multicall = MagicMock(side_effect=tokens.evm_inquirer.multicall)
-
-    tokens.detect_tokens(False, [addr1, addr2])
-    assert tokens.evm_inquirer.multicall.call_count == 0  # don't use multicall for detection
-    result, token_usd_prices = tokens.query_tokens_for_addresses([addr1, addr2])
+    detection_result = tokens.detect_tokens(False, [addr1, addr2, addr3])
+    # WETH is owned by the proxy, but should be returned in the proxy owner address
+    assert A_WETH in detection_result[addr3][0]
```

Comment:
This is what the assertion message is for

```suggestion 
    assert A_WETH in detection_result[addr3][0], 'WETH is owned by the proxy, but should be returned in the proxy owner address',  # noqa: E501
```